idea/src/META-INF/plugin.xml

```diff
@@ -2710,6 +2715,15 @@
                      language="kotlin"
     />
 
+    <localInspection implementationClass="org.jetbrains.kotlin.idea.intentions.ConvertPairConstructorToToFunctionInspection"
+                     displayName="Convert Pair constructor to 'to' function"
+                     groupPath="Kotlin"
+                     groupName="Style issues"
+                     enabledByDefault="true"
+                     level="WEAK WARNING"
```

Comment:
Not quite sure but I'd prefer `INFORMATION` level here. I think these two kinds of syntax are more or less equivalent. May be `to` looks a bit nicer, but not in every case.